sdk/identity/azure-identity/inc/azure/identity/client_certificate_credential.hpp

```diff
@@ -24,6 +24,26 @@ namespace Azure { namespace Identity {
   namespace _detail {
     class TokenCredentialImpl;
 
+#ifdef WIN32
```

Comment:
Use the platform.hpp header to define the various platform constants rather than depending on `WIN32` to be defined.

Reply:
Done.